docs: Blog tutorial editorial review, add a little content love (#582)
cosmos/sdk-tutorials: blog/tutorial/01-index.md

This is a common pattern in the SDK: users make changes to the store by broadcasting [messages](https://docs.cosmos.network/master/building-modules/messages-and-queries.html#messages). Both CLI commands and HTTP requests create messages that can be broadcasted in order for state transition to occur.

## Define the Message to Create a Post

> Define `NewMsgCreatePost` in a new file you should create as `x/blog/types/messages_post.go`.

Reviewer: ```suggestion
To define the message to create a new post, create a new file named  `NewMsgCreatePost` in `x/blog/types/messages_post.go`.
```

Reviewer: we're creating a new file?

> After being broadcast, the messages are processed by an important part of the application, called [**handlers**](https://docs.cosmos.network/master/building-modules/handler.html).

Reviewer: ummmm I think handlers are deprecated now, Cosmos SDK modules use Msg services instead https://docs.cosmos.network/master/building-modules/msg-services.html 

what changes do we need here?

	return &sdk.Result{Events: ctx.EventManager().ABCIEvents()}, nil
}
```

> After creating a post object with creator, ID and title, the message handler calls `k.CreatePost(ctx, post)`. “k” stands for [Keeper](https://docs.cosmos.network/master/building-modules/keeper.html), an abstraction used by the SDK that writes data to the store. Define the `CreatePost` keeper function in a new `keeper/post.go` file.

Reviewer: ```suggestion
After creating a post object with creator, ID, and title, the message handler calls `k.CreatePost(ctx, post)`. “k” stands for [Keeper](https://docs.cosmos.network/master/building-modules/keeper.html), an abstraction used by the SDK that writes data to the store. Define the `CreatePost` keeper function in a new `keeper/post.go` file.
```

Reviewer: we'll have other handler-related updates here since the handler is deprecated

Reviewer: let's continue to move in the direction of "you" as the audience instead of the collective first person, see https://www.notion.so/allinbits/Tutorials-62b31701f5124b7989f2cdaa01350a08 and https://ddbeck.com/second-person-is-ok/ (lots of opinions on audience!)

    return
}

```

> `CreatePost` creates a key by concatenating a post prefix with an ID. If you look back at how our store looks, you’ll notice keys have prefixes, which is why `post-0bae9f7d-20f8-4b51-9d5c-af9103177d66` contained the prefix `post-` . The reason for this is you have one store, but you might want to keep different types of objects in it, like posts and users. Prefixing keys with `post-` and `user-` allows you to share one storage space between different types of objects.

Reviewer: ```suggestion
`CreatePost` creates a key by concatenating a post prefix with an ID. 

Look at the store and you’ll notice that keys have prefixes. For example, the `post-0bae9f7d-20f8-4b51-9d5c-af9103177d66` has the prefix `post-` . The prefix identifies the different types of objects in the store, like posts and users. Prefixing keys with `post-` and `user-` allows you to share one storage space between different types of objects.
```

## Add the Prefix for a Post

To define the post prefix add the following code:

```go
// x/blog/types/keys.go
package types

const (
	// Other constants...
	// PostPrefix is used for keys in the KV store
  	PostKey= "Post-value-"
	PostCountKey= "Post-count-"
)
```

## Add the Codec

> Finally, `store.Set(key, value)` writes our post to the store.

Reviewer: ```suggestion
Finally, `store.Set(key, value)` writes the post to the store.
```

> Two last things to do is tell our [encoder](https://docs.cosmos.network/master/core/encoding.html#amino) how the `MsgCreatePost` is converted to bytes.

Reviewer: ```suggestion
Now tell the [encoder](https://docs.cosmos.network/master/core/encoding.html#amino) how to convert  `MsgCreatePost` to bytes.
```

> Two last things to do is tell our [encoder](https://docs.cosmos.network/master/core/encoding.html#amino) how the `MsgCreatePost` is converted to bytes.

Reviewer: ```suggestion
Now tell the [encoder](https://docs.cosmos.network/master/core/encoding.html#amino) how to convert  `MsgCreatePost` to bytes.
```

```

> ## Launch the Application

Reviewer: ```suggestion
## Launch the App
```

Reviewer: let's use app consistently 
see the word list https://developers.google.com/style/word-list#letter-a 
> app
> In general, use app instead of application when referring to programs for end users, especially in the context of mobile or web software. In some contexts, such as enterprise software, it's OK to use application to convey a sense of greater complexity. Use application in standard phrases such as application programming interface.
>

Now you are ready to build and start the app and create some posts.

> To launch the application run:

Reviewer: ```suggestion
To launch the app, run:
```

Run the following command to create a post:

```sh
blogd tx blog create-post "My first post" "This is a post\!" --from=alice
```

> “My first post” is a title for our post and `--from=alice` tells the program who is creating this post. `alice` is a label for your pair of keys used to sign the transaction, created by the initialization script located within the `/Makefile` previously. Keys are stored in `~/.blogd`.

Reviewer: ```suggestion
“- My first post” is a title for our post
- `--from=alice` is the author of the post
- `alice` is a label for your pair of keys that is used to sign the transaction
- Key pairs were created earlier by the initialization script in the `/Makefile` 
- Keys are stored in `~/.blogd`
```

Reviewer: I wasn't clear on what was created earlier... the key pairs or the transaction or ?? make sure I don't introduce errors here, teach me!

> Key pairs were created earlier by the initialization script in the `/Makefile`

> After running the command and confirming it, you will see an object with “txhash” property with a value like `4B7B68DEACC7CDF3243965A449095B4AB895C9D9BDF0516725BF2173794A9B3C`.

Reviewer: ```suggestion
After you run and confirm the command, an object is created with “txhash” property with a value like `4B7B68DEACC7CDF3243965A449095B4AB895C9D9BDF0516725BF2173794A9B3C`.
```

> To verify that the transaction has been processed, open a browser and visit the following URL (make sure to replace `4B7B6...` with the value of your txhash but make sure to have the `0x` prefix):

Reviewer: ```suggestion
To verify that the transaction has been processed, open a web browser and visit the following URL:
```

```
http://localhost:26657/tx?hash=0x4B7B68DEACC7CDF3243965A449095B4AB895C9D9BDF0516725BF2173794A9B3C
```

> 

Reviewer: ```suggestion
In the URL, retain the `0x` prefix but replace `4B7B6...` with the value of your txhash:
```

> Also check out a basic block overview at

Reviewer: ```suggestion
You can check out a basic block overview in a web browser:
```
